cmd/contour/serve.go

```diff
@@ -1209,6 +1211,7 @@ func (s *Server) getDAGBuilder(dbc dagBuilderConfig) *dag.Builder {
 			GlobalRateLimitService:        dbc.globalRateLimitService,
 			PerConnectionBufferLimitBytes: dbc.perConnectionBufferLimitBytes,
 			SetSourceMetadataOnRoutes:     true,
+			GlobalCircuitBreakerDefaults:  dbc.globalCircuitBreakerDefaults,
```

Comment:
Still need to pass `dbc.globalCircuitBreakerDefaults` into the `IngressProcessor`, above.

Reply:
thanks nice catch, I fixed the `cluster_test` and I had a false sense of confidence, I codified this in `serve_test` as well

Reply:
fixed in 4151035aa07d44ff5b162b52dd73935da4f7cda2 + added tests